src/ComposerScript/RsaOaep256AlgorithmInstaller.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ComposerScript;
+
+use Composer\Script\Event;
+
+class RsaOaep256AlgorithmInstaller {
```

Comment:
Please provide some basic explanation what the installer does - it's pretty clear, but for consistency.

Reply:
gree/jose library is missing content encryption algorithm RSA-OAEP-256 which is mandatory for encrypted 
    REST communication with hyperwallet app. That is because gree/jose library is based on phpseclib/phpseclib, 
    which has only RSA-OAEP support, but we can configure RSA-OAEP to become RSA-OAEP-256. RsaOaep256AlgorithmInstaller copies 
    JWE.php file with RSA-OAEP-256 configuration to the gree/jose library, so it can support RSA-OAEP-256 algorithm

Reply:
Thanks Andrii, I meant in the class :)